README.md

```diff
@@ -1,37 +1,101 @@
 # pg-backup-api
-A server that provides an HTTP API to interact with Postgres backups
+
+A server that provides an REST API to interact with Barman.
```

Comment:
The reason for leaving "Postgres backups" is because we never intended for this to be a barman REST API, but one we could also integrate with backrest some day.
Not sure if that will ever be the case